docs/layers/monitoring/datadog/datadog.mdx

```diff
@@ -84,118 +90,170 @@ Golden Signals are a form of telemetry that applies to anything with throughput.
+      review quality.</dd>
+    <dt>Force Merged by Admins (bypassing approvals)</dt>
+    <dd>The number of pull requests merged by admins without the usual approval process. This metric highlights instances
+      where standard review procedures were skipped.</dd>
+  </dl>
+---
+#### Code Quality
+
+    <dl>
+      <dt>Code Coverage</dt>
+      <dd>The percentage of your codebase covered by automated tests. This metric helps ensure that your code is thoroughly
+        tested.</dd>
+      <dt>Static Code Analysis</dt>
+      <dd>The number of issues found by static code analysis tools. This metric helps identify potential bugs and security
+        vulnerabilities.</dd>
+      <dt>Defect Escape Rate</dt>
+      <dd>The percentage of defects found in production that weren't caught by automated tests. This metric indicates the
+        effectiveness of your testing strategy.</dd>
+    </dl>
 
-- Pull Request: Discussions # Comments
+---
+#### Customer Experience
+
+  <dl>
+    <dt>Page Load Time</dt>
+    <dd>The time it takes for a web page to load. This metric is crucial for ensuring a positive user experience.</dd>
+    <dt>Browser Interaction Time</dt>
+    <dd>The time it takes for a user to interact with a web page. This metric helps gauge the responsiveness of your
+      application.</dd>
+    <dt>Server Error Rate</dt>
+    <dd>The percentage of server errors returned to users. This metric indicates the reliability of your backend
+      infrastructure.</dd>
+    <dt>JS Errors</dt>
+    <dd>The number of JavaScript errors encountered by users. This metric helps identify issues that impact the user
+      experience.</dd>
+  </dl>
+
+### Service-Oriented View
+
+#### What is a Service?
 
-- Pull Request: Force Merged by Admins (bypassing approvals)
+- A service is anything that warrants having an SLO
 
-- Code Quality
+- A service has a contract with internal parties or external parties to provide a “service” with certain guarantees. Guarantees with a contractual commitment are SLAs.
 
-- Static Code Analysis :question_mark:
+- We Should be able to look at a business service and drill down to the alerts affecting that service
 
-- Defect Escape Rate
+#### Questions to ask:
+- What are the services we provide? e.g. API
 
-- Customer Experience: Page Load Time
+- Who are the consumers of those services? e.g. internal users or customers
 
-- Customer Experience: Browser Interaction Time
+- Who is the support team of those services?
 
-- Customer Experience: Server Error Rate
 
-- Customer Experience: JS Errors
+### Error Budget
 
-- Log Events
+The Error Budget determines risk tolerance. It is usually a calculation of the remaining time in a given period that the SLO can be violated before the SLA is violated.
 
-### **Service-Oriented View**
+<img src="/assets/refarch/image-20211006-041558.png" width={"40%"} style={{"display": "block", "marginLeft": "auto", "marginRight": "auto" }} /><br/>
```

Comment:
I wanted the image centered and not full size. Thoughts on how we want to standardize this?